app/cli/internal/action/attestation_status.go

```diff
@@ -83,8 +86,8 @@ func (action *AttestationStatus) Run(ctx context.Context, attestationID string)
 	workflowMeta := att.GetWorkflow()
 
 	res := &AttestationStatusResult{
+		AttestationID: workflowMeta.GetWorkflowRunId(),
```

Comment:
We may want to review this naming at some point. Sometimes it is confusing and I understand workflow run vs attestation